src/backend/gl/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
-name = "gfx_device_gl"
-version = "0.14.0"
+name = "gfx_backend_gl"
```

Comment:
Nice to see it renamed! :D Although a crates.io search for "gfx" might look weird.. Is there a way to remove a crate from public search like a full crate yank ?

Reply:
I would prefer a large `[DEPRECATED]` sign in the crate description text for the `gfx_device_` and `gfx_window_` crates.